src/main/kotlin/app/batch/HBaseWriter.kt

```diff
@@ -28,41 +29,41 @@ class HBaseWriter(private val connection: Connection) : ItemWriter<DecompressedS
     override fun write(items: MutableList<out DecompressedStream>) {
         items.forEach {
```

Comment:
This doesn't seem to meet the AC "Failure of one file stops the whole process". Here, failure of one **record** stops the **file** as far as I can see, but subsequent files are processed. If so, just need to confirm with Nick this is what we want to happen. Also, we should have an integration test for the specific desired behaviour either way.

Reply:
If one record fails it is logged and we press on (line 59 HBaseWriter)